wasmplugin/config.go

```diff
@@ -32,5 +33,13 @@ func parsePluginConfiguration(data []byte) (pluginConfiguration, error) {
 		config.rules = append(config.rules, value.String())
 		return true
 	})
+
+	identifier := jsonData.Get("identifier")
+	if identifier.Exists() {
+		config.identifier = identifier.String()
+	} else {
+		config.identifier = "default"
```

Comment:
Let's leave the default as empty string and have the metric label only include it when it's set.

Reply:
should we keep identifier exist when empty?
for example, if empty
```
waf_filter.tx.interruptions_ruleid=1234_identifier=_phase=phase
```

or should we remove it?
```
waf_filter.tx.interruptions_ruleid=1234_phase=phase
```

Reply:
I would remove it.